src/rgw/rgw_kms.cc

```diff
@@ -80,8 +80,8 @@ class ZeroPoolAllocator {
 	return r;
     }
     void* Realloc(void* p, size_t old, size_t nw) {
-	void *r = nullptr;
-	if (nw) r = malloc(nw);
+    if (!nw) return 0;
```

Comment:
spacing is not alligned with code otherwise LGTM

Reply:
Sorry for my mistake, fixed.